### PR TITLE
Game winner layout styles

### PR DESCRIPTION
**Pull request by beccarothwell**

### CODE REVIEW

**Review by jennysivapalan — COMMENTED** [review]
I get this error on running the branch. Is there a command I need to run?

Screenshot 2024-01-10 at 08 43 58

File: client/src/components/Winner/GameWinner.tsx [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
-          </div>
-        )}
-      </>
-    );
-  } else {
-    const playersString = players.map((p) => p.name).join(", ");
+  const singleWinner = players.length === 1;
+  const winnerNameString = singleWinner
+    ? players[0].name
+    : players.map((p) => p.name).join(", ");
```
Comment (jennysivapalan) [inline client/src/components/Winner/GameWinner.tsx:14]: Does this all need to change? The previous way was readable too?

File: client/src/components/Winner/GameWinner.tsx [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
-        <div data-testid="players-drawn">
-          {playersString} have drawn! Have another game to catch them all!
-        </div>
-      </>
-    );
-  }
+  return (
+    <>
+      {players[0] &&
+        (singleWinner ? (
```
Comment (jennysivapalan) [inline client/src/components/Winner/GameWinner.tsx:19]: Is these needed? I think the previous way is a bit easier to read.

File: client/src/components/GameBoard/GameBoard.tsx [hunk 3]
```diff
@@ -168,17 +169,13 @@ const GameBoard: React.FC<GameBoardProps> = ({ pack }) => {
...
-      resetGame();
     }
   };
   const buttonText =
     gameStatus === "READY"
       ? "Start Game"
       : gameStatus === "ROUND_READY"
       ? "Next Round"
-      : gameStatus === "FINISHED"
-      ? "Play Again?"
```
Comment (jennysivapalan) [inline client/src/components/GameBoard/GameBoard.tsx:181]: Just to check -  this is gone as it's in the popup now?

Comment (beccarothwell) [reply client/src/components/GameBoard/GameBoard.tsx:181]: Yeah this was moved to the winner popup

**Comment by beccarothwell** [comment]
> I get this error on running the branch. Is there a command I need to run?
> 
> Screenshot 2024-01-10 at 08 43 58

Fixed this

File: client/src/components/Winner/GameWinner.tsx (hunk 2 above)
Comment (jennysivapalan) [reply client/src/components/Winner/GameWinner.tsx]: resolving this as we have the convo about which is related

File: client/src/components/Winner/GameWinner.tsx (hunk 1 above)
Comment (beccarothwell) [reply client/src/components/Winner/GameWinner.tsx]: Screenshot 2024-01-10 131253
Screenshot 2024-01-10 131446
I don't think they're too different? But I can see how the if else is maybe more readable than the ternary? So I can switch it to that if preferred?

Comment (jennysivapalan) [reply client/src/components/Winner/GameWinner.tsx]: I'm fine with either - I'm less of a fan of the `const singleWinner = players.length === 1;` and this bit below `return (
    <>
      {players[0] &&
        (singleWinner ? (`  feels complex. 
        
        but both look solved with the above?

**Review by jennysivapalan — APPROVED** [review]

**Pull request merged**